docs/zh/latest/architecture-design/debug-mode.md

```diff
@@ -23,7 +23,7 @@ title: Debug Mode
 
 ### 基本调试模式
 
-设置 `conf/config.yaml` 中的 `apisix.enable_debug` 为 `true`，即可开启基本调试模式。
+设置 `conf/debug.yaml` 中的 `enable_debug` 为 `true`，即可开启基本调试模式。
```

Comment:
Hi, for those who using apisix versions before 2.9.0, this document may be confused

Reply:
We can add a note for the version before 2.10.0?